http-agent/src/main/java/org/motechproject/http/agent/domain/HTTPActionAudit.java

```diff
@@ -17,7 +17,7 @@
     @Field
     private String request;
```

Comment:
Btw. can we also make the request a TEXT type? 255 characters is not that much and I can easily imagine someone complaining about it soon.